drop-in/src/main/java/com/adyen/checkout/dropin/ui/action/ActionComponentDialogFragment.kt

```diff
@@ -33,18 +34,13 @@ class ActionComponentDialogFragment : DropInBottomSheetDialogFragment() {
     private var _binding: FragmentGenericActionComponentBinding? = null
```

Comment:
I have another cool way to do this if you're interested you can fing it [here](https://github.com/etman55/android-house/blob/main/app/src/main/java/com/atef/clubhouse/base/extension/ViewBinding.kt) 
and you will be using it like this 
```
private val binding by viewBinding(FragmentMainBinding::bind)
```

Reply:
That's pretty cool! I know how to make what you linked even better. That's for another PR though

Reply:
I'll work on this PR as I don't have much of stuff to do right now actually.